### DISCUSSION TITLE
Request example of the plasmoid configuration

### DISCUSSION THREAD

**Discussion by Gertjuh** [korapp/plasma-homeassistant#58] [General]
Hi ,
Thanks for this application. Nice work, and I would like to use it.
In the ReadMe it says: "_Create Long Lived Access Token on your Home Assistant Profile page (https://<your_ha_address>/profile) and put it in the plasmoid configuration_". After searching for a while I finally found the place where the files are stored. But I have no clue where to "_put it in the plasmoid configuration_". Can you please give some guidance in what file to put the key and in what format? An example would be nice. 
Thanks!

**Comment by korapp** [comment]
Hi,
First of all, there is no need to do any file manipulation.

Plasmoid has a configuration UI like any other plasmoid, hence no detailed description.
![hap_conf_example](https://github.com/user-attachments/assets/16b7d346-6d8a-43cb-bb90-8babd3831d35)

Simply place the plasmoid in the desired location and click "Configure..." (or right-click and then "Configure Home Assistant..." if you placed it in the panel). You will see fields for Home Assistant URL and token.

Actually, I just noticed that tokens have been moved to the subpage https://<ha_address>/profile/security, so the hint

 link below the token field will not take you directly there. You will find it in the security tab of the profile though.

Then simply add the desired entities in the Items configuration tab.

Hope this helps.

**Comment by Gertjuh** [comment]
Made no progress. The picture above seems to be the ' web-browser-extended'. To install that I need 'kpackagetool6' which I can't find anywhere.

Installation (https://github.com/Hou-Rui/web-browser-extended)
Download the latest release web-browser-extended.plasmoid. Run kpackagetool6 --type Plasma/Applet --install web-browser-extended.plasmoid to install the applet.

**Comment by korapp** [comment]
This plasmoid does not require any other dependency than websocket (see table in requirements).
**The preferred and easiest way to install is to use Plasma Discover or KDE Get New Stuff and search for Home Assistant.**
Then you don't need kpackagetool6.
If you are new to KDE Plasma, check out the plasmoid installation: https://userbase.kde.org/Plasma/Installing_Plasmoids
